pages/api/paybutton/index.ts

```diff
@@ -33,6 +33,9 @@ export default async (req: any, res: any): Promise<void> => {
         case RESPONSE_MESSAGES.INVALID_BUTTON_DATA_400.message:
           res.status(400).json(RESPONSE_MESSAGES.INVALID_BUTTON_DATA_400)
           break
+        case RESPONSE_MESSAGES.WALLET_ID_NOT_PROVIDED_400.message:
+          res.status(400).json(RESPONSE_MESSAGES.WALLET_ID_NOT_PROVIDED_400)
+          break
```

Comment:
where can this `WALLET_ID_NOT_PROVIDED_400` be thrown?

Reply:
Indeed it can't yet, it's on the next PR.

I put it here to easy the review of the next one, because it brings changes in a lot of files and this is roughly the stuff that can be taken out without breaking anything.

Reply:
ok, nice :)